src/openrct2/PlatformEnvironment.cpp

```diff
@@ -234,7 +250,6 @@ const u8string PlatformEnvironment::FileNames[] = {
     u8"objects.idx",                             // CACHE_OBJECTS
     u8"tracks.idx",                              // CACHE_TRACKS
     u8"scenarios.idx",                           // CACHE_SCENARIOS
-    u8"Data" PATH_SEPARATOR "mp.dat",            // MP_DAT
```

Comment:
Since you didn’t remove MP_DAT from the enum, doesn’t this mean the rest of the list now all maps incorrectly?

Reply:
I thought I did :/